nibabel/viewers.py

```diff
@@ -61,6 +61,9 @@ def __init__(self, data, affine=None, axes=None, title=None):
         title : str or None, optional
             The title to display. Can be None (default) to display no
             title.
+        vlim : array-like or None, optional
```

Comment:
Won't that be true for any not-default limits that we set?  I mean, don't we need to apply the heuristic for any limits?